UnitTests/MockScriptMessage.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import WebKit
+
+class MockScriptMessage: WKScriptMessage {
+
+    var _body: Any?
+    var _name: String?
```

Comment:
Can we make these private?

Reply:
Sure can! done here: https://github.com/braintree/popup-bridge-ios/pull/56/commits/a665b15341762a1dd78d9be19e242e29a42bbcb5